CHANGELOG.md

```diff
@@ -65,13 +65,13 @@
   ```
   - `auth.signIn()` is split into different methods
   ```dart
-    // Magic link signin
+    // Magic sign-in link
```

Comment:
`Magic link` is the term we typically refer to this feature as Supabase, so could we do this?

```suggestion
    // Magic link sign-in
```